center_x: [-1.2198541164398193, 0.18006634712219238, 0.9186580181121826]
center_y: [2.004964828491211, 0.4965808391571045, 5.227665901184082]
center_z: [1.8851757049560547, 3.733076572418213, 3.278716564178467]
frame_id: map
is_occupied: [0, 1, 1]
size_x: [1.5098600387573242, 1.5098600387573242, 1.5098600387573242]
size_y: [3.58927059173584, 3.58927059173584, 3.58927059173584]
size_z: [1.078592300415039, 1.078592300415039, 1.078592300415039]
